apps/test/unit/code-studio/pd/professional_learning_landing/landingPageTest.js

```diff
@@ -357,25 +385,35 @@ describe('LandingPage', () => {
     const fetchStub = jest
       .spyOn(window, 'fetch')
       .mockClear()
-      .mockResolvedValue({
-        ok: true,
-        json: () => Promise.resolve({workshops_as_organizer: [TEST_WORKSHOP]}),
+      .mockImplementation(args => {
+        if (args.includes('workshops_user_enrolled_in')) {
+          return Promise.resolve({ok: true, json: () => []});
+        } else if (args.includes('workshops_as_organizer_for_pl_page')) {
```

Comment:
TIL how to have a mock implementation split the resolved promise based on what args it's called with!

Reply:
This took me a while to figure out 😅